mutmut/__main__.py

```diff
@@ -980,12 +986,10 @@ def run_stats_collection(runner, tests=None):
 
     os.environ['MUTANT_UNDER_TEST'] = 'stats'
     os.environ['PY_IGNORE_IMPORTMISMATCH'] = '1'
-    with CatchOutput(show_spinner=True, spinner_title='running stats') as output_catcher:
-        collect_stats_exit_code = runner.run_stats(tests=tests)
-        if collect_stats_exit_code != 0:
-            output_catcher.dump_output()
-            print(f'failed to collect stats. runner returned {collect_stats_exit_code}')
-            exit(1)
+    collect_stats_exit_code = runner.run_stats(tests=tests)
```

Comment:
You can use outputcatcher.stop() to stop it from swallowing output. This is a better method for debugging I find.

Although, the real solution is to add a flag or config to turn it off globally I guess...

Reply:
Thank you for the Tip